torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -41,52 +41,71 @@ const std::string shape_compute_functions =
           assert len(out) == 2
           return [out[0], out[1]]
 
+        # TODO: maybe make it customary that extra arguments are unused ?
+        # TODO: return self directly
+        def unary_two_unused_inputs(self: List[int], inp0: Any, inp1: Any):
+          out: List[int] = []
+          for elem in self:
+            out.append(elem)
```

Comment:
Nit: You could use out.extend(self) instead of this loop.

Reply:
right now i am limiting it to operators which we can partially optimize out. in future prs im hoping to add more coverage to list ops and add hard errors when you use an operator that is not symbolically optimizable